frontend/pubspec.yaml

```diff
@@ -1,4 +1,4 @@
-name: frontend
+name: dip
```

Comment:
@garrysjh are well all using package name as frontend instead? if so can we stick to frontend @ptlinh1803